packages/compression/index.ts

```diff
@@ -1,4 +1,6 @@
-import { Injectable, MiddlewareFunction, NestMiddleware } from '@nestjs/common';
+import { resolve } from 'url';
```

Comment:
this should not be here

Reply:
You are right, auto import from my vscode, fixed in the last commit (b088894)